psignifit/psigniplot.py

```diff
@@ -329,8 +329,9 @@ def plot_prior(result: Result,
         prior_cdf = np.cumsum(prior_val * prior_w)
         q25_index = np.argmax(prior_cdf > 0.25)
         q75_index = np.argmax(prior_cdf > 0.75)
+        prior_mean = np.sum(prior_x * prior_val)/np.sum(prior_val)  # ? copied from MATLAB
```

Comment:
Please remove the comment. I think the division by `np.sum(prior_val)` is unnecessary, because the prior functions should be normalized already. Let's leave it in, for consistency with Matlab

Reply:
comment removed!

I tried without the division but it did not work, to be honest I don't really understand the operation, I just copied the line from matlab. It seems to work.

The black dot should be the 'prior mean', it is also described  in the paper like that.